api/controllers/main.js

```diff
@@ -21,11 +21,17 @@ module.exports = {
 
     const siteDisplayEnv = config.app.app_env !== 'production' ? config.app.app_env : null;
 
+    const frontendConfig = {
```

Comment:
explicitly add the necessary front end config values to the index page's context